bin/cli.js

```diff
@@ -8,13 +8,28 @@ const common = require('metarhia-common');
 
 const metaschema = require('..');
 
-const cwd = process.cwd();
-const apiFile = path.resolve(cwd, process.argv[2]);
+let apiFile = path.resolve(process.cwd(), process.argv[2]);
+const packageFile = path.join(apiFile, 'package.json');
+
+if (fs.statSync(apiFile).isDirectory() && fs.existsSync(packageFile)) {
+  apiFile = path.join(apiFile, JSON.parse(fs.readFileSync(packageFile)).main);
```

Comment:
I don't think there is any need to check if you were provided a directory with the `package.json` since you are using `require()` later which already supports loading a directory. Correct me if I'm wrong.